Storage/Storage/Model/GeneralStoreSettings.swift

```diff
@@ -63,6 +63,17 @@ public struct GeneralStoreSettings: Codable, Equatable, GeneratedCopiable {
         self.firstInPersonPaymentsTransactionsByReaderType = firstInPersonPaymentsTransactionsByReaderType
         self.selectedTaxRateID = selectedTaxRateID
     }
+
+    public func erasingSelectedTaxRateID() -> GeneralStoreSettings {
```

Comment:
Should we use "erase" for the function name?

```suggestion
    public func eraseSelectedTaxRateID() -> GeneralStoreSettings {
```

Reply:
Here I used a pattern we had before in our code: when we return a copy of the object with some parameters changed we used the suffix ing: see here https://github.com/woocommerce/woocommerce-ios/blob/0a0e591ba576e77efafe03df6f75f2afc63ea48a/Storage/Storage/Model/GeneralAppSettings.swift#L137